aspnetcore/web-api/advanced/api-conventions.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using web API conventions
+author: pranavkm
+description: Learn about web API conventions
+ms.author: pranavkm
```

Comment:
Add the following metadata above this line, so that the doc is only available for ASP.NET Core 2.2+:

```
monikerRange: '>= aspnetcore-2.2'
```